src/triage/cli.py

```diff
@@ -22,6 +42,46 @@ def __call__(self, args):
         print(CONFIG_VERSION)
 
 
+@Triage.register
+class Audition(Command):
+    """ Audition command to run or validate the config file
+    """
+    def __init__(self, parser):
+        parser.add_argument(
+            '-c', '--config',
+            type=argparse.FileType('r'),
+            default='audition_config.yaml',
+            help="config file for audition",
+        )
+        parser.set_defaults(
+            directory=None,
+            validate=True,
+        )
+
+    @cachedproperty
+    def runner(self):
+        db_url = self.root.db_url
+        dir_plot = self.args.directory
+        config = yaml.load(self.args.config)
+        db_engine = create_engine(db_url)
+        print(db_engine)
+        return AuditionRunner(config, db_engine, dir_plot)
+
+    def __call__(self, args):
+        self['run'](args)
+
+    @cmdmethod('-d', '--directory', default=os.getcwd(), help="directory to store the result plots from audition")
+    @cmdmethod('-v', '--validate', action='store_true', help="validate first")
```

Comment:
you set the `validate` default to `True` in `Audition.__init__`, so this option isn't useful – if you indeed want to validate by default, make this –

    '--no-validate', action='store_false', dest='validate', …